tools/bin/commands/release

```diff
@@ -109,10 +106,16 @@ release::run() {
     # Create the image for the upgrade
     create_upgrade_docker_image "$topdir" "$release_version"
 
-    # Create the operator image
+    # Create the operator image binaries
     update_image_versions "$topdir" "$release_version" "$moving_tag"
     "$topdir/install/operator/build.sh" --operator-build docker --image-build docker --image-name "syndesis/syndesis-operator" --image-tag "$release_version"
 
+    # Prepare binaries for release
+    prepare_binaries "${topdir}/install/operator/dist" "${topdir}/install/operator/releases"
+
+    # Release the binaries
+    publish_artifacts "${topdir}/install/operator/releases" "$release_version"
```

Comment:
I'd move towards the end, there's no guarantee that the tags, docker pushes and Maven central staging repository release will happen. No point in announcing a release without those happening before.